wdfn-server/waterdata/templates/base_plain.html

```diff
@@ -20,10 +20,19 @@
                 'FIM_ENDPOINT': '{{ config.FIM_ENDPOINT }}',
                 'STATIC_URL': '{{ config.STATIC_ROOT }}',
                 'TIMESERIES_AUDIO_ENABLED': '{{ config.TIMESERIES_AUDIO_ENABLED}}' === 'True' ? true : false,
-                'WEATHER_SERVICE_ROOT': '{{ config.WEATHER_SERVICE_ROOT }}'
+                'WEATHER_SERVICE_ROOT': '{{ config.WEATHER_SERVICE_ROOT }}',
+                'DATE_TIME_FORMAT_TIMEZONE': "{{ 'scripts/date-time-format-timezone-complete-min.js' | asset_url }}"
             };
         </script>
         <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+        <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en"></script>
+        <script type="text/javascript">
+            if (!!document.documentMode) {
+                document.write("<script src=" + ${CONFIG.DATE_TIME_FORMAT_TIMEZONE} + "><\/script>");
```

Comment:
This throws an error in the browser. For basic string concatentation you don't surround the variable by ${...}, just specify the variable. The other syntax is used for template literals which you can't use directly in the browser anyways.

Reply:
Why does `DATE_TIME_FORMAT_TIMEZONE` need to be on the config object? Could the script tag just directly reference `{{ 'scripts/date-time-format-timezone-complete-min.js' | asset_url }}`?

Reply:
I don't think so. The javascript executes the string concatentation so would treat that as a straight string. @skaymen can comment, but I believe he tried it that way initially.

Reply:
Right, but I mean skip the string concatenation altogether and just render with Jinja:

```
    document.write("<script src="{{ 'scripts/date-time-format-timezone-complete-min.js' | asset_url }}"><\/script>");
```

Reply:
@danielnaab you are right. @skaymen, try the following: 
`document.write('<script src="{{ "scripts/date-time-format-timezone-complete-min.js" | asset_url }}"' + '<\/script>');`
Then you won't need the variable.